### ISSUE TITLE
Airgeddon problem with hashcat decryption

### ISSUE THREAD

**Issue by GhostDog76** [v1s1t0r1sh3r3/airgeddon#123]
Hi guys newbie on hacking and cracking wifi.....
running aroung to try and hack some wifi i found the following problem with hashcat decryption,

"The charset to use is: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789!"#$%&'()*+,-./:;<=>?@[\]^_`{|}~] 

Starting decrypt. When started, press [Ctrl+C] to stop... 
Press [Enter] key to continue...
hashcat (v4.0.1) starting... 

OpenCL Platform #1: The pocl project 
==================================== 

>>> /tmp/hctmp.hccapx: No such file or directory <<<

* Device #1: pthread-AMD A6-6310 APU with AMD Radeon R4 Graphics, 2048/5203 MB allocatable, 4MCU 

Started: Fri Apr  6 10:11:12 2018 
Stopped: Fri Apr  6 10:11:12 2018 
Press [Enter] key to continue..."

i know for sure my lil laptop ain't the top notch for hacking,
but i'm guessing the following error has nothing to do with my laptop performance

>>> /tmp/hctmp.hccapx: No such file or directory <<<

to be sure wasn't a glitch or something i went to check if the file really missing while airgeddon was running,but when i went to /tmp folder i found the file right there along with hctmp.out.
can anybody somebody tell me how to fix it?
coz deep down i really like that tool i mean airgeddon.

anyway i am running parrot 3.11 with the last system wide upgrade made 2 3 days ago,last version of airgeddon 8.01 "or something" ran the update just yesterday from within airgeddon..
coz on system wide upgrade never popped up.

thanks for any help comming in my way..
cheers folks and happy password cracking to ya'll

**Comment by OscarAkaElvis** [comment]
I saw it too. It sounds like a problem in Parrot. It seems in last version hashcat is broken. I tried the same version of airgeddon on Kali and it worked.

I remember some time ago was the opposite, Kali had the broken hashcat and was working on Parrot but it seems now everything changed. You must know that to run hashcat using CPU is a "tricky" thing... if you are using a virtual machine there are a lot of problems.

For sure, if you try it on other platform it will work. I already reported this to parrot staff. We hope they will fix it on future versions. Closing.

**Comment by v1s1t0r1sh3r3** [comment]
Pretty strange my friend... I did some tests. First tested on Kali with hashcat 4.2.1 and everything worked fine. Then I did "apt update && apt dist-upgrade" on my Kali and rebooted. After restarting, hashcat was 5.0.0 and I tried a hashcat dict attack over a captured file and it worked like a charm. Here is a screenshot (marked in red the hashcat version and the "Cracked" status)

![imagen](https://user-images.githubusercontent.com/17899365/48735295-46893e00-ec48-11e8-8e0a-fb995a87f952.png)


Probably you have some problen on your installation. Anyway, it's not related to airgeddon.
